src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -219,7 +221,13 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
 
       allocate(gradDensityEdge(nVertLevels, nEdges), &
                dzdxEdge(nVertLevels, nEdges), &
-               drhoDzTopOfEdge(nVertLevels, nEdges))
+               drhoDzTopOfEdge(nVertLevels, nEdges), &
```

Comment:
@vanroekel in the current code, `dzdxEdge` and `drhoDzTopOfEdge` are allocated and set to zero but never used. I can take them out, but can you double check that not using them was intentional, rather than a copy and paste error?

Reply:
@mark-petersen this was a copy-paste error   Thanks for catching that!